add portStrictMode option (#319)
GoogleChrome/chrome-launcher: README.md

  // Default: an available port is autoselected
  port: number;

>   // (optional) When a port is specified but no Chrome is found at that port:
>   // * if this option is true and there is no Chrome with that debug port, throw an error
>   // * else, launch a new Chrome with that debug port

Reviewer: ```suggestion
  // (optional) When `port` is specified *and* no Chrome is found at that port,
  // * if `false` (default), chrome-launcher will launch a new Chrome with that port.
  // * if `true`, throw an error
```
